### PR TITLE
Added a method to count select options 

### PR DESCRIPTION
**Pull request by maxneuvians**
Shows "All benefits for you to consider" when there is no selection. Addresses #315

### CODE REVIEW

File: components/BB.js [hunk 1]
```diff
@@ -175,11 +175,22 @@ export class BB extends Component {
     this.setState({ sortByValue: event.target.value });
   };
 
+  countSelection = () => {
+    const reducer = (acc, obj) => acc + (Object.values(obj)[0] == null ? 0 : 1);
+    let count = Object.values(this.props.selectedEligibility).reduce(
```
Comment (sastels) [inline components/BB.js:180]: Neat!

File: __tests__/components/BB_test.js [hunk 2]
```diff
@@ -203,4 +203,71 @@ describe("BB", () => {
...
+        selectedEligibility: {
+          patronType: elegibilityPathsFixture[0].patronType
+        }
+      });
+      expect(
+        mounted_BB()
+          .instance()
+          .countSelection()
+      ).toEqual(2);
+    });
```
Comment (sastels) [inline __tests__/components/BB_test.js:271]: You’ve drunk the testing Koolaid I see. Nice.

File: components/BB.js [hunk 3]
```diff
@@ -175,11 +175,22 @@ export class BB extends Component {
...
+      reducer,
+      0
+    );
+    return count + Object.values(this.props.selectedNeeds).length;
+  };
+
   countString = (x, t) => {
-    switch (x) {
-      case 0:
+    switch (true) {
```
Comment (sastels) [inline components/BB.js:188]: Seems like a strange way to use a switch statement (Ie vs an if-else). Is the a common thing to do in JavaScript?

**Review by sastels — APPROVED** [review]
Minor comment, LGTM

File: __tests__/components/BB_test.js (hunk 2 above)
Comment (maxneuvians) [reply __tests__/components/BB_test.js:271]: I figure being remote the tests need to be good :)

File: components/BB.js (hunk 3 above)
Comment (maxneuvians) [reply components/BB.js:188]: No JS - but in my mind makes it more readable. Happy to discuss.

Comment (sastels) [reply components/BB.js:188]: nope, leave it as it is, I don't have strong opinions either way. 👍

**Pull request merged**